app/src/main/java/com/hedvig/app/feature/embark/passages/previousinsurer/askforprice/AskForPriceInfoActivity.kt

```diff
@@ -0,0 +1,92 @@
+package com.hedvig.app.feature.embark.passages.previousinsurer.askforprice
+
+import android.app.Activity
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.animation.ExperimentalAnimationApi
+import androidx.compose.material.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import com.hedvig.app.R
+import com.hedvig.app.ui.compose.composables.TopAppBarWithBack
+import com.hedvig.app.ui.compose.theme.HedvigTheme
+
+class AskForPriceInfoActivity : ComponentActivity() {
```

Comment:
Should this use `BaseActivity`? If not, I believe we might be loading the incorrect locale here.

Reply:
Good catch, fixing